modules/extensions/src/views/lite/components/debugger/components/Intents.tsx

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash'
-import React, { Fragment } from 'react'
+import React from 'react'
 
+import ContentSection from '../../../../../../../../src/bp/ui-shared-lite/ContentSection'
```

Comment:
Could we import then export this component from ui-shared ? The goal of ui-shared-lite was only to make some components available for the webchat, which shouldn't load ui-shared. 

I really hate having a ton of ../../../../../ ....

Reply:
will find a solution to the shitty import in a different PR, but agreed